src/util/dev/server.ts

```diff
@@ -329,11 +347,70 @@ export default class DevServer {
       }
     }
 
+    const apiFiles = await getApiFiles(this.cwd, this.output);
+
+    if (apiFiles.length > 0) {
```

Comment:
Should this be `if (apiFiles && apiFiles.length > 0) {` like on line 364?

Reply:
Nope, it will always return an array, so I've removed line 364 👍